vignettes/cqc_directory.Rmd

```diff
@@ -0,0 +1,146 @@
+---
+title: "cqc_directory dataset"
+author:
+- '[Martine Wauben](mailto:martine.wauben@dhsc.gov.uk)'
+date: "04/10/2020"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{cqc_directory dataset}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+options(tidyverse.quiet = TRUE)
+```
+
+This vignette explains how to use the `cqc_directory` dataset in R, and also
+details where it comes from and how it is generated.
+
+The data is sourced from [the Care Quality Commission](https://www.cqc.org.uk/about-us/transparency/using-cqc-data)
+and its archive is available on [Google Drive](https://drive.google.com/drive/folders/1Y6V6r-q2l4lJYKuZL0DXw25VDUTH6L4N). 
```

Comment:
is this a personal google drive folder or is this from CQC? this may be worth including in the data/ folder or somewhere else in this project's repository if it is something that has been generated from the raw data provided by the CQC.